relay-profiling/src/android.rs

```diff
@@ -58,6 +59,9 @@ struct AndroidProfile {
 
     #[serde(default = "AndroidProfile::default")]
     profile: AndroidTraceLog,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    measurements: Option<HashMap<String, Vec<Measurement>>>,
```

Comment:
We don't need to make this an `Option`, we can just `default` and call `HashMap::is_empty` if we don't want to serialize it.